.github/workflows/check.yml

```diff
@@ -112,7 +112,21 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Install packages
-        run: sudo apt-get install -yq cppcheck 
+        run: |
+          sudo apt-get install -yq wget build-essential
+
+          # Download cppcheck-2.4.1 source
+          wd=`pwd`
+          cd ..
```

Comment:
Because the checkout action deletes the working directory, it should be located in the other directory.